arch/arch.go

```diff
@@ -0,0 +1,48 @@
+// package arch contains mappings between the Golang architecture and
+// the RPM architecture used by Fedora CoreOS and derivatives.
+package arch
+
+import "runtime"
+
+type mapping struct {
+	rpmArch string
+	goArch  string
+}
+
+// If an architecture isn't defined here, we assume it's
+// pass through.
+var translations = []mapping{
+	{
+		rpmArch: "x86_64",
+		goArch:  "amd64",
+	},
+	{
+		rpmArch: "aarch64",
+		goArch:  "arm64",
+	},
+}
+
+// CurrentRpmArch returns the current architecture in RPM terms.
+func CurrentRpmArch() string {
+	return RpmArch(runtime.GOARCH)
+}
+
+// RpmArch translates a Go architecture to RPM.
+func RpmArch(goarch string) string {
+	for _, m := range translations {
+		if m.goArch == goarch {
+			return m.rpmArch
+		}
+	}
+	return goarch
```

Comment:
Maybe this isn't a concern, but would the "pass through" assumption be potentially deceiving? i.e. one would assume whatever comes out of the `RpmArch` function is definitely an RPM Arch.

Reply:
It's definitely a concern!  The downside is that we'd need to maintain an exhaustive list here of all architectures to validate it, I didn't think that was worth the maintenance overhead.

Reply:
Sounds fair :)

Reply:
A good example is I notice the coreos-assembler code is already missing [RISC-V](https://en.wikipedia.org/wiki/RISC-V) and...honestly I'm not sure if that matches or not.  OK looks like [the Go arch](https://github.com/golang/go/issues/27532) is `riscv64` and the RPM one is (time passes while I dig in [links from here](https://fedoraproject.org/wiki/Architectures/RISC-V)) that too!  Hooray.